.github/workflows/azure-static-web-apps-green-glacier-0ed825f1e.yml

```diff
@@ -0,0 +1,67 @@
+name: Azure Static Web Apps CI/CD
+
+on:
+  workflow_dispatch:
+  push:
+    branches: 
+      - main

```

Comment:
You can either add the assignment 3 branch to this, or can remove the branches component entirely to make this happen every time you push